views/forms/EditChartForm.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState, useEffect } from 'react'
+import { compose } from 'redux'
+import { connect } from 'react-redux'
+import { withStyles } from '@material-ui/core/styles'
+import Button from '@material-ui/core/Button'
+
+import Form from './Form'
+import BarChartFields from './BarChartFields'
+
+import { chartStyles } from '../styles/chart_styles'
+import { getChartForm } from '../fe-utils/fetchUtil'
+
+const ChartForm = ({ classes, handleSubmit, user, graph }) => {
+  const [formValues, setFormValues] = useState({})
+  const [load, setLoading] = useState(true)
+
+  useEffect(() => {
+    getChartForm(graph.chart_id).then((res) => {
+      setFormValues(res.data)
+      setLoading(false)
+    })
```

Comment:
do we need a catch here in case the request fails?

Reply:
There's a catch in the fetchUtil function